.drone.yml

```diff
@@ -81,6 +81,85 @@ steps:
           - main
     depends_on:
       - build & push latest
+
+  - name: build & push libreoffice
```

Comment:
Can you please deprecate the old repository please?

Reply:
its gone